docs/loading/Flink-connector-starrocks.md

```diff
@@ -1,173 +1,647 @@
+| sink.ignore.update-before         | No           | true              | Supported since version 1.2.8. Whether to ignore `UPDATE_BEFORE` records from Flink when loading data to primary key table. If false, will treat the record as a delete operation to StarRocks table.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| sink.properties.*                 | No           | NONE              | The parameters that are used to control Stream Load behavior.  For example, the parameter `sink.properties.format` specifies the format used for stream load, such as CSV or JSON. For a list of supported parameters and their descriptions, see [STREAM LOAD](https://docs.starrocks.io/en-us/latest/sql-reference/sql-statements/data-manipulation/STREAM%20LOAD).                                                                                                                                                                                                                                                                                                                                 |
+| sink.properties.format            | No           | csv               | The format used for stream load. The connector will transform each batch of data to the format before sending them to StarRocks. Valid values: csv and json.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/af8f76a8-4032-4d03-ac50-94e0b4ee6acd)
-> The Flink connector